share/letsencrypt/dehydrated-confconsole.cron

```diff
@@ -1,30 +1,33 @@
-#!/bin/sh
+#!/bin/bash
 
-export PATH=$PATH:/usr/sbin
+export PATH="$PATH:/usr/sbin"
 
 CERT=/etc/ssl/private/cert.pem
 RENEW=2592000 # seconds to cert expiry to try renew: 2592000 = 30 days
 LOG=/var/log/confconsole/letsencrypt.log
-LOG_INFO=  # set to always log certificate expiry check
 DEHYDRATED=/usr/lib/confconsole/plugins.d/Lets_Encrypt/dehydrated-wrapper
 ARG="--force"
 
 log() {
-    echo -n "[$(date "+%Y-%m-%d %H:%M:%S")] cron: ${1}" >> $LOG
+    echo "[$(date "+%Y-%m-%d %H:%M:%S")] cron: $*" >> $LOG
 }
 
 exit_code=0
-check_cert_expire=$($(which openssl) x509 -checkend $RENEW -noout -in $CERT) \
+
+cert_expire=$(/usr/bin/openssl x509 -checkend $RENEW -noout -in $CERT) \
     || exit_code=$?
-msg="${CERT}: ${check_cert_expire} within $(( $RENEW / 60 / 60 / 24 )) days"
+log "${CERT}: ${cert_expire} within $(( RENEW / 60 / 60 / 24 )) days"
 
-if [ "$exit_code" -eq 0 ]; then
-    [ -z "$LOG_INFO" ] \
-        || log "${msg}. Nothing to do.\n"
+if [[ "$exit_code" -eq 0 ]]; then
+    log "Nothing to do."
 else
-    log "${msg} (or has already). Attempting renewal.\n"
-    $DEHYDRATED $ARG
-    if [ $? -ne 0 ]; then
-        log "ERR: $(basename $DEHYDRATED) exited with a non-zero exit code.\n"
+    exit_code=0
+    log "Attempting renewal."
+    log "$(DEHYDRATED $ARG 2>&1)" || exit_code=$?
+    if [[ $exit_code != 0 ]]; then
```

Comment:
Just a style thing, you use ``!=`` here, but ``-eq`` above. Doesn't really matter because in both cases the check is strict equality/inequality between integers, but style wise it might be nicer to be consistent.

Reply:
addressed in new PR